open_academy/models/__init__.py

```diff
@@ -1 +1,3 @@
 from . import course 
+from . import session
+from . import inherited_partner
```

Comment:
Rename the model file as `partner.py`

```suggestion
from . import partner
```